index.html

```diff
@@ -46,7 +46,7 @@ <h1 class="heading1">
         you see and have a project you need coded, don’t hestiate to contact me.
       </p>
 
-      <p class="suggegiion-comment">LET’S CONNECT</p>
+      <header>class="suggegiion-comment">LET’S CONNECT</header>
```

Comment:
Looks like you confused `<h2>` with `<header>`, make sure to make this small change and your project will be approved! :wink:

Reply:
Noted,let me that.

Reply:
Hello Kakalanp,I hope you are well.I made  the changes you requested me to do,My project was not approved.